zone/mob.cpp

```diff
@@ -3714,10 +3714,19 @@ bool Mob::HateSummon() {
 			// probably should be like half melee range, but we can't get melee range nicely because reasons :)
 			new_pos = target->TryMoveAlong(new_pos, 5.0f, angle);
 
-			if (target->IsClient())
+			if (target->IsClient()) {
 				target->CastToClient()->MovePC(zone->GetZoneID(), zone->GetInstanceID(), new_pos.x, new_pos.y, new_pos.z, new_pos.w, 0, SummonPC);
-			else
-				target->GMMove(new_pos.x, new_pos.y, new_pos.z, new_pos.w);
+			}
+			else {
+				bool set_new_guard_spot;
+				bool target_is_client_pet;
```

Comment:
Why not initialize these variables with their values from here if you’re doing so a line below?

Reply:
Trying to make the lines wrap less.  Want it changed?

Reply:
We can just format it in a way that they're separate lines. Similar to how I did this.
https://github.com/ProjectEQ/projecteqquests/commit/a0462dfb389c4cd0f3a1b3873fb8fbb955378241